### ISSUE TITLE
Add issue template

### ISSUE THREAD

**Issue by sydneywalcoff** [sydneywalcoff/app-tracker#20]
# Overview

As a developer, I would like to be able to submit issues quickly and easily

## Action Items

- [ ] add github issue template to the repo
- [ ] unblock #53 and move to new issue ready for approval

## Resources

- [github issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository)
